bot/requests/modules/planning/core.js

```diff
@@ -28,7 +28,7 @@ async function coreProcess(author, dates = [], search = null, target = null){
         ical = await getUserICS(namedotsurname);
     }catch(err){
         if(global.debug) console.log(err);
-        return "Une erreur s'est prduite à la récupération de l'emplois-du-temps. Veuillez réessayer.\n (*" + err + "*)";
+        return "Une erreur s'est produite à la récupération de l'emplois-du-temps. Veuillez réessayer.\n (*" + err + "*)";
```

Comment:
l'emplois-du-temps => l'emploi du temps

Reply:
This is fixed [here](https://github.com/ISEN-Promo-Pole-developpement-Discord/Sigma-VI/pull/46/commits/3a6c981554a3642d537c1f4e467639ce0ef3288e).